mkdocs_monorepo_plugin/edit_uri.py

```diff
@@ -56,7 +56,7 @@ def __load_page_config_file(self, file):
 
     root_docs_dir = self.__get_root_docs_dir()
     root_repo_url = self.config.get('repo_url')
-    root_edit_uri = self.config.get('edit_uri', '')
+    root_edit_uri = self.config.get('edit_uri', '') or ''
```

Comment:
Interesting, I expected the default value to be the second parameter passed to the "get" method (an empty string) in case an "edit_uri" property didn't exist in the configuration object

Reply:
Yeah, it's kinda unintuitive!  Seems to be that `dict.get()` will return the second value in case of `edit_uri` not being found in `self.config` at all.  But if `self.config.edit_uri` is explicitly set to `None`, then it will return `None` and ignore the second argument.

Apparently starting in mkdocs v1.4.0, instead of not defining it at all, `edit_uri` is always defined, but is set to `None` when it doesn't exist in the config.